### ISSUE TITLE
Home Assistant Assist support

### ISSUE THREAD

**Issue by saya6k** [matt8707/ha-fusion#287]
In Home Assistant Lovelace, you can see Assist button on top right corner.

I wish there are ways to interact with Assist is also possible in Fusion. Option to choose between use microphone and speaker / chat would be nice.

**Comment by matt8707** [comment]
Yes, I have experimented with this, and only Chrome seems to work with voice.
